spec/system/task_spec.rb

```diff
@@ -1,24 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project){create(:project)}
+  let(:task){create(:task, project_id: project.id)}
```

Comment:
Factoryに`association`を定義するとproject_idの指定は不要となります。
```suggestion
  let(:task){ create(:task) }
```

Reply:
fed8518 Factoryの定義で、taskにassociationの設定を付与し、project_idの指定を外しました。